include/CppUTest/StandardCLibrary.h

```diff
@@ -1,8 +1,10 @@
 
+/* Must include this first to ensure the StandardC include in CppUTestConfig still happens at the right moment */
+#include "CppUTestConfig.h"
```

Comment:
Because it makes the code uglier, and because StandardCLibrary.h seems like an implementation detail that shouldn't be getting included directly in user test code in the first place.  (I don't have strong feelings on this, but I do think it'd be tidier that way.)